megaavr/extras/LowPower.md

```diff
@@ -0,0 +1,635 @@
+# LowPower.cpp
```

Comment:
Let's add a paragraph or two at the start of this document describing the features of the low-power library and its goals at a high level.